### ISSUE TITLE
Zoom glitches out if scrolling during map coasting

### ISSUE THREAD

**Issue by MoffKalast** [MoffKalast/Sailwind-Map#8]
When the map is panned and released it seems to retain some velocity and coasts further for a bit, depending on the speed of the mouse. If any kind of mouse scroll zoom is attempted it'll try to zoom and get reset back, typically several times. There should be some kind of check that stops the map movement in case a zoom attempt is made, but I'm not sure if ArcGIS has the callback for that event.

**Comment by hassankhalil33** [comment]
I think this is done purposely to stop accidental scroll while moving. you think stopping the map pan and instead allow the user to zoom would be preferable?

**Comment by MoffKalast** [comment]
I think it would be more responsive that way, yep.  Iirc it mainly stems from replacing the stock scrolling (which randomly gets stuck before reaching min/max zoom as it doesn't clamp the value) with this custom bit https://github.com/MoffKalast/Sailwind-Map/blob/8b70f958c9493e1e8f3e5f2d601403a97d3ad5bc/js/main.js#L410

**Comment by hassankhalil33** [comment]
yes I did some experimenting yesterday. The default scroll without the bit u mentioned works as intended but wont feel smooth and is a bit too fast, also it wont center the scroll on the mouse. The only workaround is to disable the coasting entirely, tho I'm not experienced in ArcGIS sadly.

**Comment by MoffKalast** [comment]
Ah yeah I think that's the same snag I hit last time I tried to fix it I think, I'm not sure if it's even possible to disable in the current version of it.
